ldi-nifi/ldi-nifi-processors/ldes-client-processor/src/test/java/be/vlaanderen/informatievlaanderen/ldes/ldi/processors/LdesClientTest.java

```diff
@@ -19,6 +17,9 @@
 import java.util.Map;
 import java.util.stream.Stream;
 
+import com.github.tomakehurst.wiremock.client.WireMock;
+import com.github.tomakehurst.wiremock.junit5.WireMockTest;
```

Comment:
Please revert this .. Normally impsort should have prevented this.. Otherwise we create and unending stream of changes on import order

Reply:
@Tomvbe at least this is what i expect of Impsort, right?

Reply:
If we're using impsort to avoid these kinds of issues and impsort is not doing what we expect, we should take a closer look at impsort. @Tomvbe ideas ?